src/test/rpc/LedgerRPC_test.cpp

```diff
@@ -1212,6 +1212,56 @@ class LedgerRPC_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testLedgerEntryDID()
+    {
+        testcase("ledger_entry Request DID");
+        using namespace test::jtx;
+        using namespace std::literals::chrono_literals;
+        Env env{*this};
+        Account const alice{"alice"};
+
+        env.fund(XRP(10000), alice);
+        env.close();
+
+        // Lambda to create a DID.
+        auto didCreate = [](test::jtx::Account const& account) {
+            Json::Value jv;
+            jv[jss::TransactionType] = jss::DIDSet;
+            jv[jss::Account] = account.human();
+            jv[sfData.jsonName] = strHex(std::string{"data"});
+            jv[sfURI.jsonName] = strHex(std::string{"uri"});
+            return jv;
+        };
+
+        env(didCreate(alice));
+        env.close();
+
+        std::string const ledgerHash{to_string(env.closed()->info().hash)};
+
+        {
+            // Request the payment channel using its index.
+            Json::Value jvParams;
+            jvParams[jss::did] = alice.human();
+            jvParams[jss::ledger_hash] = ledgerHash;
+            Json::Value const jrr = env.rpc(
```

Comment:
Are we using the DID format `did:xrpl:0:...` to perform any CRUD operation? It appears that we use Account(`alice` in this piece of code) and Type (`DID`).

The implementation does adhere to the spec. Is the DIDID meant to be a universal identifier without any usages inside ledger itself?

Reply:
No, that format is used in the universal resolver, not in `ledger_entry`. It's unnecessary in `ledger_entry`.